pycbc/inference/models/relbin.py

```diff
@@ -317,16 +306,16 @@ def init_from_frequencies(self, h00, fbin_ind, ifo):
         )
 
         # store low res copy of fiducial waveform
-        self.h00_sparse[ifo] = h00.copy().take(fbin_ind)
+        self.h00_sparse[ifo] = h00[ifo].copy().take(fbin_ind)
 
         # compute summary data
         logging.info(
             "Calculating summary data at frequency resolution %s Hz",
             self.df[ifo],
         )
 
-        a0, a1 = self.summary_product(self.data[ifo], h00, bins, ifo)
-        b0, b1 = self.summary_product(h00, h00, bins, ifo)
+        a0, a1 = self.summary_product(data, h00[ifo], bins, ifo)
+        b0, b1 = self.summary_product(h00[ifo], h00[ifo], bins, ifo)
```

Comment:
So the values here should then still be the same?

Reply:
Yes, that's right. They are unchanged. It's switching where the time shift is away from the parts we might want to compare on the fly. The time shift makes it difficult for example to compare if two waveforms are similar (at least quickly).